xds/googledirectpath/googlec2p.go

```diff
@@ -77,13 +76,7 @@ var (
 )
 
 func init() {
-	resolver.Register(c2pResolverBuilder{
-		scheme: c2pScheme,
-	})
-	// TODO(apolcyn): remove this experimental scheme before the 1.52 release
-	resolver.Register(c2pResolverBuilder{
-		scheme: c2pExperimentalScheme,
-	})
+	resolver.Register(c2pResolverBuilder{scheme: c2pScheme})
```

Comment:
Let's remove the `scheme` from the `Builder` now since we don't need it to be changeable.

Reply:
Done.